mteb/tasks/Retrieval/multilingual/PosIRRetrieval.py

```diff
@@ -0,0 +1,591 @@
+  author = {Ziyang Zeng and Dun Zhang and Jiacheng Li and Panxiang Zou and Yuqing Yang},
+  eprint = {2505.13950},
+  primaryclass = {cs.IR},
+  title = {An Empirical Study of Position Bias in Modern Information Retrieval},
+  url = {https://arxiv.org/abs/2505.13950},
+  year = {2025},
+}
+""",
+}
+
+_COMMON_DESCRIPTION_SUFFIX = (
+    "Its passages are sampled directly from this subset. Queries are synthetically generated by LLMs, covering a range of types "
+    "including 'what', 'how', 'why', 'where', 'when', as well as summarization and boolean questions."
+)
+
+
+# load data function
+def _load_data(
+    path: str,
+    langs: list,
+    split: str,
+    industry_domain: str,
+    cache_dir: str = None,
+    revision: str = None,
+):
+    queries = defaultdict(dict)
+    corpus = defaultdict(dict)
+    relevant_docs = defaultdict(dict)
+
+    for lang in langs:
+        # load queries data
+        data = load_dataset(
+            path,
+            data_files=f"{lang}/{industry_domain}/queries.parquet",
+            cache_dir=cache_dir,
+            revision=revision,
+        )
+
+        queries[lang][split] = {item["_id"]: item["text"] for item in data["train"]}
+
+        # get corpus
+        data = load_dataset(
+            path,
+            data_files=f"{lang}/{industry_domain}/corpus.parquet",
+            cache_dir=cache_dir,
+            revision=revision,
+        )
+        corpus[lang][split] = {item["_id"]: item for item in data["train"]}
+
+        # get relevant_docs
+        data = load_dataset(
+            path,
+            name=f"{lang}-{industry_domain}",
+            split=split,
+            cache_dir=cache_dir,
+            revision=revision,
+        )
+        # the qrels contain all labels, we only use the selected labels
+        relevant_docs[lang][split] = {
+            item["query-id"]: {item["corpus-id"]: 1}
+            for item in data
+            if item["query-id"] in queries[lang][split]
+        }
+
+    return corpus, queries, relevant_docs
+
+
+# Common load_data method for all PosIR classes
+def _common_load_data(self, **kwargs):
+    if self.data_loaded:
+        return
+    self.corpus, self.queries, self.relevant_docs = _load_data(
+        path=self.metadata.dataset["path"],
+        langs=self.hf_subsets,
+        split=self.metadata.eval_splits[0],
+        industry_domain=self._industry_domain,
+        cache_dir=kwargs.get("cache_dir", None),
+        revision=self.metadata.dataset["revision"],
+    )
+    self.data_loaded = True
+
+
+# 31 test sets
+class PosIRAccommodationCateringHotelRetrieval(MultilingualTask, AbsTaskRetrieval):
+    _industry_domain = "accommodation_catering_hotel"
+    metadata = TaskMetadata(
+        **_POSIR_METADATA,
+        name="PosIRAccommodationCateringHotelRetrieval",
+        domains=["Non-fiction"],
+        description=(
+            "This test set is based on the accommodation_catering_hotel subset of the BAAI/IndustryCorpus2 dataset. "
```

Comment:
I need a description of what the corpus and queries are. This is better, but it requires the user to know what the accommodation_catering_hotel contains